CONTRIBUTING.md

```diff
@@ -6,36 +6,72 @@ For major changes, please open an issue to discuss what you would like to change
 As of writing this document, the project is still in a pre-1.0 state.
 This means that the API is not yet stable and breaking changes may occur frequently.
 
-## Pull Requests
+## Prerequisites
+
+- [`rust`](https://www.rust-lang.org/tools/install)
+- [`docker`](https://docs.docker.com/engine/install/)
+  - You will need this to test the CI/CD pipelines locally or to use Earthly to run various project commands.
+  - You may need to start the docker daemon before running any commands. You can do this with `sudo systemctl start docker`.
+  - You may need to add your user to the `docker` group to run docker commands without `sudo`. You can do this with `sudo usermod -aG docker $USER`.
+- [`hermit`](https://cashapp.github.io/hermit/usage/get-started/)
```

Comment:
This is mentioned in the link so you may not need to mention it here, but I did have to manually add to my path. I'm guessing most new additions to the research group will be more computer literate than I was when I joined, but I would've had _no idea_ how to add something to path when I joined the group. Also, even now, I didn't know if I should add '/home/username/bin' or '/home/username/bin/hermit' to my path. Maybe this will be obvious to other people or need not be dealt with here, just thought I'd mention it.